scripts/upgrade

```diff
@@ -123,28 +122,11 @@ if [ "$upgrade_type" == "UPGRADE_APP" ]
 then
   ynh_script_progression --message="Upgrading source files..." --weight=2
 
-  # Create a temporary directory
-  tmpdir="$(mktemp -d)"
-
-  # Backup the config file and local plugins in the temp dir
-  cp -a "$final_path/config/config.json" "$tmpdir/config.json"
-  cp -ar "$final_path/plugins" "$tmpdir/plugins"
-
-  # Remove the app directory securely
-  ynh_secure_remove --file="$final_path"
-
   if [ "$version" = "Enterprise" ]; then
-    ynh_setup_source --dest_dir="$final_path" --source_id="enterprise"
+    ynh_setup_source --dest_dir="$final_path" --source_id="enterprise" --keep="$final_path/plugins $final_path/config/config.json"
   elif [ "$version" = "Team" ]; then
-    ynh_setup_source --dest_dir="$final_path" --source_id="$architecture"
+    ynh_setup_source --dest_dir="$final_path" --source_id="$YNH_ARCH" --keep="$final_path/plugins $final_path/config/config.json"
```

Comment:
Does "--keep" merge the new content in the plugins directory with the previous one? (we want to both update built-in plugins and keep our custom plugins)

Reply:
mmh, you are right, this is just preserving the plugins folder from being updated...